core/src/main/resources/i18n/displayStrings.properties

```diff
@@ -605,7 +606,7 @@ portfolio.pending.unconfirmedTooLong=Deposit transactions on trade {0} are still
 
 portfolio.pending.step1.waitForConf=Wait for blockchain confirmations
 portfolio.pending.step2_buyer.startPayment=Start payment
-portfolio.pending.step2_seller.waitPaymentSent=Wait until payment has been sent
+portfolio.pending.step2_seller.waitPaymentStarted=Wait until payment has started
```

Comment:
This line needs updated (not rebased correctly?).

Reply:
This line should not be changed from current master. Revert.

Reply:
This is current master: https://github.com/haveno-dex/haveno/blob/34b79e779b565d2102a4f32210658aea5c83217a/core/src/main/resources/i18n/displayStrings.properties#L608

<img width="670" alt="image" src="https://user-images.githubusercontent.com/13068859/221903321-8fd174db-bc6c-4f43-ac34-a539645528ff.png">

Your PR should not make these changes.